### PR TITLE
New methods: manifest, validator_info

### PR DESCRIPTION
**Pull request by mDuo13**
Documents the new methods added by https://github.com/ripple/rippled/pull/3197 and cleans up the admin methods landing (#765)

### CODE REVIEW

File: content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md [hunk 1]
```diff
@@ -27,13 +27,20 @@ Use these methods to manage log levels and other data, such as ledgers.
...
-* **[`connect`](connect.html)** - Force the rippled server to connect to a specific peer.
 * **[`ledger_accept`](ledger_accept.html)** - Close and advance the ledger in stand-alone mode.
 * **[`stop`](stop.html)** - Shut down the rippled server.
-* **[`validation_seed`](validation_seed.html)** - Temporarily set key to be used for validating.
 
+## [Peer Management Methods](peer-management-methods.html)
+
+Use these methods to manage the server's connections in the peer-to-peer XRP Ledger network.
+
+* **[`connect`](connect.html)** - Force the rippled server to connect to a specific peer.
```
Comment (LoisRP) [inline content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md]: rippled - should this be `rippled`

File: content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/manifest.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+# manifest
+[[Source]](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/Manifest.cpp "Source")
+
+The `{{currentpage.name}}` method reports the current "manifest" information for a given validator public key. The "manifest" is the public portion of that validator's configured token.
+
+_The `{{currentpage.name}}` method is an [admin method](admin-rippled-methods.html) that cannot be run by unprivileged users._
```
Comment (LoisRP) [inline content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/manifest.md:6]: unprivileged users - Again, probably reading out of context, but is it clear what an unprivileged user is?

Comment (mDuo13) [reply content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/manifest.md:6]: This phrasing is used consistently throughout the docs. It basically means anyone who's not connected as an admin. The top of the linked admin method page links some information about how to connect as an admin (or not).

File: content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
-Admin commands are available only if you connect to `rippled` on a host and port that the `rippled.cfg` file identifies as admin. By default, the commandline client uses an admin connection. For more information on connecting to `rippled`, see [Getting Started with the rippled API](get-started-with-the-rippled-api.html).
+Admin commands are available only if you connect to `rippled` on a host and port that the `rippled.cfg` file identifies as admin. By default, the commandline client uses an admin connection. For more information on connecting to `rippled`, see [Getting Started with the `rippled` API](get-started-with-the-rippled-api.html).
 
 
 ## [Key Generation Methods](key-generation-methods.html)
 
 Use these methods to generate and manage keys.
 
-* **[`validation_create`](validation_create.html)** - Generate keys for a new rippled validator.
+* **[`validation_create`](validation_create.html)** - Generate keys for a new `rippled` validator.
```
Comment (nbougalis) [inline content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md]: We should point out that validator tokens are the preferred solution here.

**Review by nbougalis — APPROVED** [review]

**Review by seelabs — COMMENTED** [review]
:+1:

**Pull request merged**